x/qgb/types/genesis.go

```diff
@@ -51,8 +53,12 @@ func validateDataCommitmentWindow(i interface{}) error {
 	val, ok := i.(uint64)
 	if !ok {
 		return fmt.Errorf("invalid parameter type: %T", i)
-	} else if val < 100 {
-		return fmt.Errorf("invalid average EVM block time, too short for latency limitations")
+	} else if val < appconsts.MinimumDataCommitmentWindow {
+		return fmt.Errorf(
```

Comment:
should we be registering this as a formal qgb error since it would occur during state logic?

Reply:
e2fadac